examples/neural_networks/plot_mnist_filters.py

```diff
@@ -34,7 +34,7 @@
 X_train, X_test = X[:60000], X[60000:]
 y_train, y_test = y[:60000], y[60000:]
 
-mlp = MLPClassifier(hidden_layer_sizes=(50,), max_iter=10, alpha=1e-4,
+mlp = MLPClassifier(hidden_layer_sizes=(100,), max_iter=40, alpha=1e-4,
```

Comment:
Why increasing the number of hidden layers?

Reply:
I asked around the sprint and nobody could explain me why there was a 50 in the example so I decided it was better to just use the default in the docs.
No other reason, I can change it =)

Reply:
This is too limit the complexity of the model to have a faster training from what I can see in the narration.